frontend/src/StartView.svelte

```diff
@@ -67,19 +79,30 @@
   import { remote_model_to_resource } from "./ofrak/remote_resource";
 
   import { onMount } from "svelte";
+  import TextDivider from "./TextDivider.svelte";
+  import FileBrowser from "./FileBrowser.svelte";
 
   export let rootResourceLoadPromise,
     showRootResource,
     resources,
     rootResource,
-    resourceNodeDataMap;
+    resourceNodeDataMap,
+    browsedFiles;
   let dragging = false,
     selectedPreExistingRoot = null,
     preExistingRootsPromise = new Promise(() => {}),
     tryHash = !!window.location.hash;
   let mouseX, selectedAnimal;
 
   async function createRootResource(f) {
+    if (
+      f.size > 1024 * 1024 * 250 &&
+      !window.confirm("Loading a large file may be slow. Are you sure?")
```

Comment:
Can the error message define what large means (1024 * 1024 * 250). This will allow a user to know what we consider "large" (they can truncate the file, etc.).